src/table/src/components/TableBody.js

```diff
@@ -26,14 +27,17 @@ const TableBody = (props :Props) => {
     data,
     headers,
     isLoading,
+    exact,
     order,
     orderBy,
     page,
     rowsPerPage,
   } = props;
 
   const sortedData = getSortedData(headers, data, order, orderBy);
-  const dataByPage :RowData[] = sortedData.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage);
+  const dataByPage :RowData[] = exact
```

Comment:
So does the `exact` prop refer to a dataset that is already paginated?

Reply:
Yes.